src/features/header-flyout/HeaderFlyout.js

```diff
@@ -30,10 +30,15 @@ class HeaderFlyout extends React.Component<Props> {
     };
 
     render() {
-        const { header, footer, flyoutButton, children, scrollRefFn, ...rest } = this.props;
+        const { header, footer, flyoutButton, children, scrollRefFn, className, ...rest } = this.props;
 
         return (
-            <Flyout closeOnClick={false} offset={HeaderFlyout.panelOffset} {...rest}>
+            <Flyout
+                closeOnClick={false}
+                offset={HeaderFlyout.panelOffset}
```

Comment:
When opened, the offset should now align the flyout vertically with the button clicked. with the caret, it was offset by a little bit. If I remember correctly, we need to shift this over as well

Reply:
<img width="500" alt="Screen Shot 2019-03-15 at 12 20 39 PM" src="https://user-images.githubusercontent.com/1571667/54456796-efd6d100-471c-11e9-9a8b-10983e53fd4e.png">